site/content/installation/installing-nic/create-license-secret.md

```diff
@@ -0,0 +1,149 @@
+
+{{< note >}}See [here](https://docs.nginx.com/solutions/about-subscription-licenses/#for-internet-connected-environments) for a list of IPs associated with F5's licensing endpoint (`product.connect.nginx.com`).{{</ note >}}
+
+## Set up the JWT
+
+{{< include "installation/download-jwt.md" >}}
+
+The JWT needs to be configured before deploying NGINX Ingress Controller. The JWT will be stored in a Kubernetes Secret of type `nginx.com/license`, and can be created with the following command.
+
+```shell
+kubectl create secret generic license-token --from-file=license.jwt=<path-to-your-jwt> --type=nginx.com/license -n <Your Namespace> 
+```
+You can now delete the downloaded `.jwt` file.
+
+{{< note >}}
+The Secret needs to be in the same Namespace as the NGINX Ingress Controller Pod(s).
+{{</ note >}}
+
+{{< include "installation/jwt-password-note.md" >}}
+### Using the NGINX Plus License Secret
+
+If using a name other than the default `license-token`, provide the name of this Secret when installing NGINX Ingress Controller:
+
+{{<tabs name="plus-secret-install">}}
+
+{{%tab name="Helm"%}}
+
+Specify the Secret name using the `controller.mgmt.licenseTokenSecretName` helm value.
+
+For detailed guidance on creating the Management block via Helm, refer to the [Helm Configuration Documentation]({{< relref "installation/installing-nic/installation-with-helm/#configuration" >}}).
+
+{{% /tab %}}
+
+{{%tab name="Manifests"%}}
+
+Specify the Secret name in the `license-token-secret-name` Management ConfigMap key.
+
+For detailed guidance on creating the Management ConfigMap, refer to the [Management ConfigMap Resource Documentation]({{< relref "configuration/global-configuration/mgmt-configmap-resource/" >}}).
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+**If you are reporting to the default licensing endpoint, then you can now proceed with [installing NGINX Ingress Controller]({{< relref "installation/installing-nic/" >}}). Otherwise, follow the steps below to configure reporting to NGINX Instance Manager.**
+
+
+### Reporting to NGINX Instance Manager {#nim}
+
+If you are deploying NGINX Ingress Controller in an "air-gapped" environment you will need to report to NGINX Instance Manager (NIM) instead of the default licensing endpoint.
+
+First, you must specify the endpoint of your NGINX Instance Manager:
+
+{{<tabs name="nim-endpoint">}}
+
+{{%tab name="Helm"%}}
+
+Specify the endpoint using the `controller.mgmt.usageReport.endpoint` helm value.
+
+{{% /tab %}}
+
+{{%tab name="Manifests"%}}
+
+Specify the endpoint in the `usage-report-endpoint` Management ConfigMap key.
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+#### Configuring SSL Certificates and SSL Trusted Certificates {#nim-cert}
+
+To configure SSL Certificates and/or SSL Trusted Certificate, a few extra steps are needed.
+
+To use Client Auth with NIM, first create a Secret of type `kubernetes.io/tls` in the Namespace that the NIC Pod(s) are in. 
```

Comment:
```suggestion
To use Client Auth with NGINX Instance Manager, first create a Secret of type `kubernetes.io/tls` in the same namespace as the NGINX Ingress Controller pods. 
```

Reply:
The user may or may not have multiple, but using plural should not create confusion and is clearer language.